shared-libs/rules-engine/src/provider-wireup.js

```diff
@@ -116,6 +116,10 @@ module.exports = {
    * @returns {Promise} To complete the transaction marking the subjectIds as dirty
    */
   updateEmissionsFor: (provider, subjectIds) => {
+    if (!subjectIds) {
```

Comment:
Nit: can we use parameter defaults here?

Reply:
done

Reply:
I've reverted this, cause the default param doesn't check for falsy, just undefined. And an empty string could still trigger the error.